packages/react/src/themes/teams/components/Dialog/dialogVariables.ts

```diff
@@ -25,7 +30,13 @@ export interface DialogVariables {
 export default (siteVariables): Partial<DialogVariables> => ({
   rootBackground: siteVariables.colors.white,
   rootBorderRadius: pxToRem(3),
-  rootWidth: '50vw',
+
+  rootWidth: '90vw',
+  rootWidthLarge: '50vw',
+  rootWidthLargeBreakpoint: '@media (min-width: 1280px)',
```

Comment:
I am kind a leaning more towards allowing users to specify only the px value here, not the whole string, but I am not sure. We should make that decision now, as this is the first PR that is adding breakpoints

Reply:
I am not sure that we should limit user there as media queries support a lot of attributes:
https://developer.mozilla.org/en-US/docs/Web/CSS/Media_Queries/Using_media_queries